src/modules/70-pipeline/components/ArtifactsSelection/ArtifactInterface.ts

```diff
@@ -126,3 +143,13 @@ export interface ArtifactImagePathTagViewProps {
   tagDisabled: boolean
   isArtifactPath?: boolean
 }
+
+export interface ACRArtifactType {
+  identifier: string
+  tag: any
+  tagRegex: any
+  tagType: TagTypes
+  repository?: any
+  subscription?: any
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9246757)
```suggestion
  subscription?: unknown
```